administration/src/components/Navigation.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import {Button, Navbar} from "@blueprintjs/core";
+
+export class Navigation extends React.Component<any, any> {
```

Comment:
We should avoid using `any`

Reply:
Well, you are right, but this code does not have any purpose other than displaying a skeleton.